src/frontend/components/CreatePlaylistDialog/CreatePlaylistDialog.jsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+
+import Dialog from "@mui/material/Dialog";
+import DialogTitle from "@mui/material/DialogTitle";
+import TextField from "@mui/material/TextField";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import { useAuth, useUser } from "../../context";
+import { addNewPlaylistService } from "../../services";
+
+function CreatePlaylistDialog({ onClose, open }) {
+  const [playlistName, setPlaylistName] = useState("");
+
+  const {
+    auth: { token },
+  } = useAuth();
+
+  const { userDispatch } = useUser();
+
+  const handleClose = () => {
+    onClose();
+  };
+
+  return (
+    <Dialog onClose={handleClose} open={open}>
+      <DialogTitle sx={{ backgroundColor: "primary.main" }}>
+        Create New Playlist
```

Comment:
The background color of this modal is not blending w the background color of your main containers, try a different shade

![image](https://user-images.githubusercontent.com/68704186/181820531-12903fe5-0d60-4998-b45e-6083f85ed3da.png)